options.js

```diff
@@ -1,58 +1,128 @@
 let page = document.getElementById('colorSelector');
```

Comment:
colorSelector would be a better name and I think you can use const here

Reply:
Changed to const